packages/eslint-config-guild-typescript/index.js

```diff
@@ -1,3 +1,7 @@
+const OFF = 0;
+const WARN = 1;
+const ERROR = 2;
```

Comment:
Not all eslint rules allow `"on"` or `"off"` syntax, but they all allow for `0, 1, 2` ... these consts turn that into a more user friendly format.